ui/app/pages/connected-accounts/index.scss

```diff
@@ -1,5 +1,7 @@
 .connected-accounts {
   &__footer {
+    padding: 0;
```

Comment:
Nit: This is fine, but it might be better to leave this padding here and delete the padding in `connected-accounts-permissions` instead. The `popover-footer` permissions seem perfectly adequate here.

They're slightly different in that there's a little less padding on the top with  `popover-footer`, but this was done intentionally, to leave a little more room at the bottom to match the curve of the corners. This is reflected in some of the modals in the Figma designs, though not all (not sure why).

Reply:
Done in: ea434da